cmd/site-controller/controller.go

```diff
@@ -403,3 +414,43 @@ func (c *SiteController) updateChecks() {
 		}
 	}
 }
+
+func (c *SiteController) policyStateWatcher(stopCh <-chan struct{}) {
+	state := false
+	period := time.NewTicker(time.Second)
+	for {
+		select {
+		case <-period.C:
+			if c.validator.Enabled() && !state {
+				state = true
+				log.Println("policy is enabled, validating permissions")
+				for _, site := range c.siteInformer.GetStore().List() {
+					// Check if ClusterRoleBinding exists for namespace
+					cm := site.(*corev1.ConfigMap)
+					c.addClusterRoleBinding(cm)
```

Comment:
router_update will add the cluster role bindings on an update, and do so regardless of whether the CRD is actually installed, is right? And in the yaml for site controller below we bind the cluster role to the site controller account unconditionally also.

I'm not clear on why we need this conditional logic here? What is the use case it is anticipating?

Reply:
@grs done.